pom.xml

```
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>gg.solarmc</groupId>
+        <artifactId>solar-parent</artifactId>
+        <version>0.6.2</version>
+    </parent>
+
+    <artifactId>SolarClans</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>16</maven.compiler.source>
+        <maven.compiler.target>16</maven.compiler.target>
+        <junit.version>5.7.0</junit.version>
+        <solarapi.version>1.2.0-SNAPSHOT</solarapi.version>
+        <loader.version>0.5.2</loader.version>
+        <dazzleconf.version>1.2.0-M2</dazzleconf.version>
+    </properties>
+
+    <distributionManagement>
+        <repository>
+            <id>solar-repo</id>
+            <url>https://mvn-repo.solarmc.gg/releases</url>
+        </repository>
+        <snapshotRepository>
+            <id>solar-repo</id>
+            <url>https://mvn-repo.solarmc.gg/snapshots</url>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <dependencies>
+        <!-- https://github.com/SolarMC-Dev/SolarPaper-API/blob/master/pom.xml -->
+        <dependency>
+            <groupId>gg.solarmc.paper</groupId>
+            <artifactId>solarpaper-api</artifactId>
+            <version>${solarapi.version}</version>
+        </dependency>
+
+        <!-- https://github.com/SolarMC-Dev/DataLoader/tree/master/mod-credits -->
+        <dependency>
+            <groupId>gg.solarmc.loader</groupId>
+            <artifactId>loader-mod-clans</artifactId>
+            <version>${loader.version}-SNAPSHOT</version>
+        </dependency>
+
+        <!-- https://github.com/SolarMC-Dev/DataLoader/tree/master/mod-credits -->
+        <dependency>
+            <groupId>gg.solarmc.loader</groupId>
+            <artifactId>loader-core</artifactId>
+            <version>${loader.version}-SNAPSHOT</version>
+        </dependency>
+
+        <!-- https://github.com/MilkBowl/VaultAPI -->
+        <dependency>
+            <groupId>com.github.MilkBowl</groupId>
```

```suggestion
            <groupId>gg.solarmc.vault</groupId>
```
And you can drop jitpack from the repositories